setup.py

```diff
@@ -0,0 +1,43 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='omega',
+    version='0.0.1',
+    packages=find_packages(),
+
+    author='Boris Yangel',
+    long_description="A number of agents (PPO, MuZero) with a \
+        Perceiver-based NN architecture that can be trained \
+            to achieve goals in nethack/minihack environments.",
+    url="https://github.com/hr0nix/omega",
+
+    install_requires=[
+        'numpy~=1.22.4',
```

Comment:
One concern that I have is that now we have two separate lists of requirements, one for docker and one for the package. Do you think it might make sense to somehow re-use this list when building a docker image?